src/scintillaeditor.cpp

```diff
@@ -644,6 +742,14 @@ bool ScintillaEditor::eventFilter(QObject* obj, QEvent *e)
 		 || e->type()==QEvent::KeyRelease
 		 ) {
 		QKeyEvent *ke = static_cast<QKeyEvent*>(e);
+		//printf("KEYPRESS. MODIFIERS: %08x\n",ke->modifiers());
+#ifdef _WIN32
+		if ((ke->modifiers() & ~Qt::KeypadModifier) == Qt::CtrlModifier|Qt::AltModifier) {
+			QKeyEvent* event = new QKeyEvent(ke->type(), Qt::Meta, ke->text());
```

Comment:
What's this change doing? It does not compile on MXE due to both Qt::CtrlModifier and Qt::Meta not being defined.